migrator/src/test/java/com/google/phonenumbers/migrator/CommandLineMainTest.java

```diff
@@ -73,4 +73,17 @@ public void createFromPath_expectSufficientArguments() {
     assertThat(p.numberInput.file).matches(TEST_FILE_INPUT);
     assertThat(p.numberInput.number).isNull();
   }
+
+  @Test
+  public void createMigrationJob_lenientFlagAndCustomRecipe_expectException() {
```

Comment:
Could you make this clear in the arg docs that these specific args cannot be used together